### ISSUE TITLE
[Accessibility] "View all pinned messages" button has poor contrasts in default light theme

### ISSUE THREAD

**Issue by juliemoynat** [mastodon/mastodon#40100]
### Steps to reproduce the problem

1. Activate the default light theme;
2. Go on a profile with pinned messages;
3. See that the "View all pinned messages" (I'm not sure of its exact name in English) has black text on blue background and is almost unreadable (contrast ratio of 2.73:1).

<img width="748" height="76" alt="" src="https://github.com/user-attachments/assets/78dccc76-6209-48f6-b0e0-080f990fd17e" />

### Expected behaviour

The button is readable.

### Actual behaviour

The button is unreadable.

### Detailed description

To fix the problem, you can just change the black text into white text (contrast ratio of 6.5:1).

### Mastodon instance

mastodon.social, eldritch.cafe…

### Mastodon version

v.4.6.4

### Browser name and version

Firefox 153

### Operating system

Windows 11

**Comment by diondiondion** [comment]
I'm not able to reproduce this; the button has a white background in our default styles (and this is what I see on mastodon.social and on your home server, too).

Could there be any user styles enabled in your web browser that might be causing this?

**Comment by juliemoynat** [comment]
Okay, I'm sorry, you're right! It's because I've added user styles because contrasts were too low before (and now it's good!).

Thank you!
